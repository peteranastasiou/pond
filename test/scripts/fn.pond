fn a() {
    print("hello");
}

fn b(a, b) {
    return a + b;
}

fn c(x) {
    if x < 0 {
        return "A1";
    }
    return if x < 1 { "B1" } else { "B2" };
}

a();

print(b(1, 3/2));

print(c(-1));
print(c(0.5));
print(c(1.5));
