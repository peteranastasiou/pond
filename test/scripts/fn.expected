hello
2.5
A1
B1
B2
